CHANGELOG.rst

```diff
@@ -17,6 +17,30 @@ This project adheres to `Semantic Versioning`_ starting with version 1.0.
 
 .. towncrier release notes start
 
+[1.10.1] - 2020-05-15
+^^^^^^^^^^^^^^^^^^^^^
+
+Improvements
+------------
+- `#5794 <https://github.com/rasahq/rasa/issues/5794>`_: Creating a ``Domain`` using ``Domain.fromDict`` can no longer alter the input dictionary.
+  Previously, there could be problems when the input dictionary was re-used for other
+  things after creating the ``Domain`` from it.
+
+Bugfixes
+--------
+- `#5617 <https://github.com/rasahq/rasa/issues/5617>`_: Don't create TensorBoard log files during prediction.
+- `#5638 <https://github.com/rasahq/rasa/issues/5638>`_: Fix: DIET breaks with empty spaCy model
+- `#5755 <https://github.com/rasahq/rasa/issues/5755>`_: Remove ``clean_up_entities`` from extractors that extract pre-defined entities.
+  Just keep the clean up method for entity extractors that extract custom entities.
+- `#5792 <https://github.com/rasahq/rasa/issues/5792>`_: Fixed issue where the ``DucklingHTTPExtractor`` component would
+  not work if its `url` contained a trailing slash.
+- `#5825 <https://github.com/rasahq/rasa/issues/5825>`_: Fix list index out of range error in ``ensure_consistent_bilou_tagging``.
+
+Miscellaneous internal changes
+------------------------------
+- #5788
```

Comment:
what happened here? this should have a description right?

Reply:
no, internal changes does not have descriptions in the changelog, that is intended I guess

Reply:
see previous releases: https://rasa.com/docs/rasa/changelog/#miscellaneous-internal-changes

Reply:
ok, why not 🤷